packages/directlinespeech/__tests__/sendSpeechActivity.js

```diff
@@ -19,7 +19,7 @@ beforeEach(() => {
 });
 
 test('should echo back when saying "hello" and "world" with enableInternalHTTPSupport set to true', async () => {
-  const { directLine, sendTextAsSpeech } = await createTestHarness({ enableInternalHTTPSupport: true });
+  const { directLine, sendTextAsSpeech } = await createTestHarness();
```

Comment:
Shouldn't this test have `enableInternalHTTPSupport` set to `true` per the description?